cpu/stm32/periph/dma.c

```diff
@@ -84,6 +85,9 @@
 #endif
 #endif /* CPU_FAM_STM32F2 || CPU_FAM_STM32F4 || CPU_FAM_STM32F7 */
 
+#define DMA_DATA_WIDTH_MASK      (0x03)
+#define DMA_DATA_WIDTH_SHIFT     (0)
```

Comment:
Why not leave this in `periph/cpu_dma.h` ?

Reply:
I moved them because they should not be part of the module's external interface. They are only used by `dma.c` to interpret the `flags` parameter passed into `dma_setup(...)`. I also believed the change I made to `dma_setup(...)`'s doxygen block might mislead someone to think they could pass `DMA_DATA_WIDTH_MASK` or `DMA_DATA_WIDTH_SHIFT` as valid flags.